copier.yaml

```diff
@@ -36,10 +41,24 @@ _tasks:
     -a use_pre_commit "no"'
 
   - "echo '==== 4/4 - Finalizing setup ===='"
-  - '"{{ python_path if python_path else _copier_python }}" post_init.py "./projects/{{ project_name }}-contracts" "./projects/{{ project_name }}-frontend"'
-  # _copier_python above is used for backwards compatibility with versions < v1.11.3 of the algokit cli
+  # Execute OS-specific file copying script (with self-destruction)
+  - command: |
```

Comment:
As discussed I don't think this is needed anymore, given 2.0.0 was launch 1 year ago.